learning_observer/learning_observer/lms_integration.py

```diff
@@ -0,0 +1,487 @@
+import json
+import recordclass
+import string
+import aiohttp
+import aiohttp.web
+
+import learning_observer
+import learning_observer.runtime
+import learning_observer.google
+import learning_observer.constants as constants
+import learning_observer.settings as settings
+import pmss
+
+pmss.register_field(
+    name="lms_api",
+    type=pmss.pmsstypes.TYPES.string,
+    description="The Canvas Base API URL",
+    required=True
+)
+
+cache = None
```

Comment:
This is used, but I can't find it being set anywhere. If it's set somewhere, that should be documented. If not, it should be removed. 

You should document why we have / need a cache, and what is stored in it.

Reply:
We reviewed. This is used for off-line demos and development. It saves calls to Google, and allows us to pretend Google exists in an off-line setting. I can masquerade as if I were on the real server. 

This doesn't work in the new code. It should be:

- Fixed
- Tested
- Properly documented

What it did was confusing in the existing code (pre-PR).